mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/persistence/KeyValueBasedPersistenceProvider.java

```diff
@@ -119,10 +122,18 @@ public class KeyValueBasedPersistenceProvider implements IMantisPersistenceProvi
 
     private final KeyValueStore kvStore;
     private final LifecycleEventPublisher eventPublisher;
+    private final Counter noWorkersFoundCounter;
 
     public KeyValueBasedPersistenceProvider(KeyValueStore kvStore, LifecycleEventPublisher eventPublisher) {
         this.kvStore = kvStore;
         this.eventPublisher = eventPublisher;
+        Metrics m = new Metrics.Builder()
+            .id("storage")
+            .addCounter("noWorkerFound")
```

Comment:
can we have a success counter along with this?

Reply:
why do we need a success counter?

Reply:
in case this one isn't emitting properly (stay nil/0) + better visibility on what gets loaded.